### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by joneshf**
This implements both steps of #3832.

I've made a review for anything contextual.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```
Comment (joneshf) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Decided to leave blank issues as an option, for now. I don't have a strong feeling about this, so I'm okay to change it if anyone wants it to happen.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve the PureScript compiler
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
```
Comment (garyb) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: Maybe just "Description" if this is the header we want in the issue itself?

Comment (joneshf) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: Sounds good.
```suggestion
## Description
```

**Comment by joneshf** [comment]
Thanks for the review!

**Pull request merged**